test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1

```diff
@@ -21,7 +22,7 @@ Describe 'ConvertTo-Json' -tags "CI" {
         $jsonFormat | Should -Match '"TestValue3": 99999'
     }
 
-	It "StopProcessing should succeed" -Pending:$true {
+    It "StopProcessing should succeed" -Pending:$true {
```

Comment:
Does this work now? Should we consider removing Pending?

Reply:
I left this for feedback - should we be worried about this or could it remove this functionality?

Reply:
I disable the test for the new experimental cmdlet but leave for old one.